src/components/Star.js

```diff
@@ -7,6 +7,7 @@ const STAR_SIZE = 40;
 
 export default class Star extends PureComponent {
   static defaultProps = {
+    baseStarImage: STAR_IMAGE,
```

Comment:
thoughts on naming this as `starImage` ?

Reply:
updated 😉